README.md

```diff
@@ -1,9 +1,8 @@
 # Practical with the Data Stewardship Wizard
 
-
-The Data Stewardship Wizard, hereon DSW, is an awareness tool in the area of research data management (RDM). 
-In this practical you'll fill in a questionnaire using the DSW. The questionnaire will guide you through key requirements in RDM making you think if and how your meet those requirements in your project.
-
+The Data Stewardship Wizard (DSW), is an awareness tool in the area of research data management (RDM).
```

Comment:
I would maybe add a bit more detailed explanation what the DSW is. Also add link to DSW official site/github and maybe logo (optional :)

Reply:
Good,  will put the link.